coordinator/persistence-cassandra-4.1/README.md

```diff
@@ -0,0 +1,27 @@
+# Persistence Cassandra 4.1
+
+This module represents the implementation of the [persistence-api](../persistence-api) for the Cassandra `4.1.x` version.
+
+Note: the initial implementation is based on the DataStax Cassandra implementation and may include code that has not been accepted into the Apache Cassandra project.
+
+## Cassandra version update
+
+The current Cassandra version this module depends on is `4.1.3`.
+In order to update to a newer patch version, please follow the guidelines below:
+
+* Update the `cassandra.version` property in the [pom.xml](pom.xml).
+* Update the `ccm.version` property (`it-cassandra-4.1` profile section) in [testing/pom.xml](../testing/pom.xml) 
+* Check the transitive dependencies of the `org.apache.cassandra:cassandra-all` for the new version.
+Make sure that the version of the `com.datastax.cassandra:cassandra-driver-core` that `cassandra-all` depends on, is same as in the `cassandra.bundled-driver.version` property in the [pom.xml](pom.xml).
+This dependency is set as optional in the `cassandra-all`, but we need it to correctly handle UDFs.
+Note that transitive dependencies can be seen on [mvnrepository.com](https://mvnrepository.com/artifact/org.apache.cassandra/cassandra-all/4.1.3).
+* Update the [CI Dockerfile](../ci/Dockerfile) and set the new version in the `ccm create` command related to 4.0.
+Note that this will have no effect until the docker image is rebuilt and pushed to the remote repository, thus creating an issue for that would be a good idea.
+* Create a separate PR for bumping the Cassandra 4 version in the Quarkus-based API integration tests on the `main` branch. Test profiles are defined in the `apis/pom.xml`.
```

Comment:
This is probably copied over as well, but I don't think this change needs to be separate PR any longer?

Reply:
I will correct this.